src/Miunie.Discord/MiunieDiscordClient.cs

```diff
@@ -14,16 +15,14 @@ public MiunieDiscordClient(IBotConfiguration botConfig)
             _botConfig = botConfig;
         }
 
-        public void Initialize()
+        public async void Initialize()
         {
-            Client = new DiscordClient(new DiscordConfiguration
+            Client = new DiscordSocketClient(new DiscordSocketConfig
             {
-                Token = _botConfig.DiscordToken,
-                TokenType = TokenType.Bot,
-                AutoReconnect = true,
-                LogLevel = LogLevel.Info,
-                UseInternalLogHandler = false
+                LogLevel = LogSeverity.Info,
```

Comment:
I also like to mention other stuff even though it defaults to that, it just makes it more clear when reading - though, that's complete personal preference and also bloat